discovery_engine/lib/src/ffi/types/document/time_spent.dart

```diff
@@ -36,7 +38,11 @@ extension TimeSpentFfi on TimeSpent {
       smbertEmbedding: EmbeddingFfi.readNative(
         ffi.time_spent_place_of_smbert_embedding(place),
       ),
```

Comment:
:art: since the order of fields changed in `TimeSpent`, it would be better to change the order of assignments here as well and align them with their declaration order. same in `writeNative()` below.